examples/src/main/java/com/mapbox/navigation/examples/core/MapboxBuildingHighlightActivity.kt

```diff
@@ -102,9 +100,9 @@ class MapboxBuildingHighlightActivity : AppCompatActivity(), OnMapLongClickListe
         MapboxNavigationConsumer<Expected<BuildingError, BuildingValue>> { expected ->
             expected.fold(
                 {
-                    LoggerProvider.logger.e(
-                        Tag("MbxBuildingHighlightActivity"),
-                        Message("error: ${it.errorMessage}")
+                    Logger.e(
```

Comment:
If we're not logging via our `LoggerProvider` proxy, we do not actually guarantee that our backend is actually set, because it's set from the `LoggerProvider`'s class init.

Reply:
```suggestion
                    LoggerProvider.logE(
```

Reply:
But `LoggerProvider` is an `object`, by that time it's already created 🤔

Reply:
`object` is just a singleton when compiled into a class, and the init block is converted to the JVM's `static` initialiazer which is only called when the class is first interacted with. So if the call to `Logger.e` would happen before `LoggerProvider` is first interacted with, our logging backend wouldn't be set yet.

Reply:
By using the functions from `LoggerProvider` only, we're ensuring that the initializer is always called before we log anything.

Reply:
Addressed. This should be ready for another round of 👀